src/librustc_interface/lib.rs

```diff
@@ -4,6 +4,7 @@
 #![feature(nll)]
 #![feature(generator_trait)]
 #![feature(generators)]
+#![feature(once_cell)]
```

Comment:
I would've expected import changes as well - is this coming from a macro or something like that? If so that should get a allow_internal_unstable annotation instead, probably

Reply:
I am not exactly sure what you are referring to, but in `src/librustc_interface/passes.rs` and `src/librustc_interface/queries.rs` the code imports `OnceCell` re-exported from `librustc_datastructures`.

Side note: there is also an explicit import (and thus a dependency on the crate) of `once_cell::sync::OnceCell` in `src/librustc_interface/util.rs` though and I don't know if that should be updated to use the impl from std.